pages/api/graphql-rest.page.ts

```diff
@@ -167,8 +167,8 @@ class MpdxRestApi extends RESTDataSource {
       newsletter: string;
       status: string;
     },
-    labelType?: ExportLabelTypeEnum,
-    sort?: ExportSortEnum,
+    labelType?: ExportLabelTypeEnum | null,
+    sort?: ExportSortEnum | null,
```

Comment:
So up in the typeDef, I have these set to be Nullable fields, and that seems to add the types ```Maybe<ExportLabelTypeEnum> | undefined```. So I add the ```?``` to the argument to resolve the undefined type. I then added the ```null``` to resolve the ```Maybe```? I'm partially confused why it's generating ```Maybe``` and ```undefined``` for the type. Maybe I'm missing something. On the client-side I'm still passing either ```ExportLabelTypeEnum``` or undefined for this.

Reply:
Oh weird.

Maybe we want this? https://www.graphql-code-generator.com/docs/generated-config/typescript#avoidoptionals Idk why this hasn't been a problem on the client side.

I also saw https://www.graphql-code-generator.com/docs/generated-config/typescript#maybevalue. Would be sorta nice to just use `undefined` everywhere but that just changes the typings, not what Apollo returns when GraphQL returns a `null` value.

Reply:
I added the ```avoidOptionals: true``` to the config but it didn't appear to change anything. Maybe it's something we could address in the eventual PR to improve the file structure/modularization of the REST -> GraphQL logic?

Reply:
Which codegen config did you add it to? The server one?

Ya we can deal with it later.

Reply:
Yeah, the server one.